src/Symfony/Bridge/Doctrine/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ CHANGELOG
    an instance of `EntityManagerInterface` should be injected instead
  * added support for `simple_array` type
  * the `DoctrineTransactionMiddlewareFactory` class has been removed
+ * changed guessing of DECIMAL to set the `input` option of `NumberType` to string
```

Comment:
Should be in a  new section for 4.3 instead.

Reply:
![](https://media.giphy.com/media/l0IykG0AM7911MrCM/giphy.gif)